vcl_templates/www.vcl.erb

```diff
@@ -198,6 +198,17 @@ sub vcl_recv {
     return(pass);
   }
 
+  if (!req.http.Cookie:ABTest-Example) {
```

Comment:
As discussed with @alexmuller, change this into: `if (req.http.Cookie !~ "ABTest-Example") {`